libs/pika/async_cuda/tests/performance/synchronize.cu

```diff
@@ -155,6 +155,10 @@ int pika_main(pika::program_options::variables_map& vm)
                 ex::transfer(ex::thread_pool_scheduler{}) | ex::then([] {}) |
                 ex::transfer(sched) | cu::then_with_stream(f) |
                 ex::transfer(ex::thread_pool_scheduler{}) | ex::then([] {}) |
+            // The compilation of this loop unrolling with llvm-amdgpu's
+            // Clang 5.3.3 is hanging.
+#if !(defined(PIKA_HAVE_P2300_REFERENCE_IMPLEMENTATION) ||                     \
+    defined(PIKA_HAVE_HIP))
```

Comment:
Please ifdef out this test or change the batch size for the full test. Right now this is misleading since it doesn't spawn as many kernels as the other tests.